specification/resources/resource-manager/Microsoft.Resources/stable/2025-03-01/resources.json

```diff
@@ -0,0 +1,7094 @@
+    },
+    "ZoneMapping": {
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The location of the zone mapping."
+        },
+        "zones": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ValidationLevel": {
+      "type": "string",
+      "description": "The level of validation performed on the deployment.",
+      "enum": [
+        "Template",
+        "Provider",
+        "ProviderNoRbac"
+      ],
+      "x-ms-enum": {
+        "name": "ValidationLevel",
+        "modelAsString": true,
+        "values": [
+          {
+            "description": "Static analysis of the template is performed.",
+            "value": "Template"
+          },
+          {
+            "description": "Static analysis of the template is performed and resource declarations are sent to resource providers for semantic validation. Validates that the caller has RBAC write permissions on each resource.",
+            "value": "Provider"
+          },
+          {
+            "description": "Static analysis of the template is performed and resource declarations are sent to resource providers for semantic validation. Skips validating that the caller has RBAC write permissions on each resource.",
+            "value": "ProviderNoRbac"
+          }
+        ]
+      }
+    },
+    "DeploymentExtensionDefinition": {
+      "type": "object",
+      "properties": {
+        "alias": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The alias of the extension as defined in the deployment template."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The extension name."
+        },
+        "version": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The extension version."
+        },
+        "configId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The extension configuration ID. It uniquely identifies a deployment control plane within an extension."
+        },
+        "config": {
+          "readOnly": true,
+          "$ref": "#/definitions/DeploymentExtensionConfig",
+          "description": "The extension configuration."
+        }
+      }
+    },
+    "DeploymentExtensionConfig": {
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/DeploymentExtensionConfigItem"
+      },
+      "description": "An extension configuration. Keys of the object are defined by an extension's configuration schema."
+    },
+    "DeploymentExtensionConfigItem": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "readOnly": true,
+          "$ref": "#/definitions/ExtensionConfigPropertyType",
+          "description": "The value type of the extension config property."
```

Comment:
Do you think config item types would be useful in deployment outputs 🤔? What’s your opinion on removing them?

Reply:
Distinguishing between secure and non-secure seems like a valid use case, so I think we should keep them.